Cannot download alpha4

Hello, there seems to be something wrong with the alpha4 npm package.
The alpha3 works.
Here is the output of my shell :

```console
PS C:\Code\test_doc> npm i -g @asciidoctor/core asciidoctor-pdf@1.0.0-alpha.3
npm WARN deprecated fsevents@1.2.9: One of your dependencies needs to upgrade to fsevents v2: 1) Proper nodejs v10+ support 2) No more fetching binaries from AWS, smaller package size
npm WARN deprecated core-js@2.6.10: core-js@<3.0 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
C:\Users\ptpl02951\AppData\Roaming\npm\asciidoctor-pdf -> C:\Users\ptpl02951\AppData\Roaming\npm\node_modules\asciidoctor-pdf\bin\asciidoctor-pdf
+ asciidoctor-pdf@1.0.0-alpha.3
+ @asciidoctor/core@2.0.3
updated 2 packages in 151.904s
PS C:\Code\test_doc> npm i @asciidoctor/core asciidoctor-pdf@1.0.0-alpha.4
npm WARN deprecated fsevents@1.2.9: One of your dependencies needs to upgrade to fsevents v2: 1) Proper nodejs v10+ support 2) No more fetching binaries from AWS, smaller package size
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t git://github.com/mathjax/MathJax-src.git
npm ERR!
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 140.82.118.4]: errno=No such file or directory
npm ERR!
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ptpl02951\AppData\Roaming\npm-cache\_logs\2019-11-18T13_48_32_984Z-debug.log
PS C:\Code\test_doc> npm i @asciidoctor/core asciidoctor-pdf@1.0.0-alpha.3
npm WARN deprecated fsevents@1.2.9: One of your dependencies needs to upgrade to fsevents v2: 1) Proper nodejs v10+ support 2) No more fetching binaries from AWS, smaller package size
npm WARN deprecated core-js@2.6.10: core-js@<3.0 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.

> puppeteer@1.15.0 install C:\Code\test_doc\node_modules\puppeteer
> node install.js

Downloading Chromium r650583 - 144.8 Mb [====================] 100% 0.0s
Chromium downloaded to C:\Code\test_doc\node_modules\puppeteer\.local-chromium\win64-650583

> core-js@2.6.10 postinstall C:\Code\test_doc\node_modules\core-js
> node postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm WARN saveError ENOENT: no such file or directory, open 'C:\Code\test_doc\package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open 'C:\Code\test_doc\package.json'
npm WARN test_doc No description
npm WARN test_doc No repository field.
npm WARN test_doc No README data
npm WARN test_doc No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ asciidoctor-pdf@1.0.0-alpha.3
+ @asciidoctor/core@2.0.3
added 249 packages from 140 contributors and audited 2397 packages in 159.209s
found 0 vulnerabilities

PS C:\Code\test_doc>
```

I believe that the root cause is:

```
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t git://github.com/mathjax/MathJax-src.git
npm ERR!
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 140.82.118.4]: errno=No such file or directory
npm ERR!
npm ERR!
npm ERR! exited with error code: 128
```

The MathJax npm package is using the following dependency:

```js
  "dependencies": {
    "mathjax-full": "git://github.com/mathjax/MathJax-src.git"
  }
```

And your corporate proxy doesn't allow to fetch a npm dependency using the git protocol.
As a workaround, you might be able to do:

```console
$ npm i -g mathjax-full@3.0.0 @asciidoctor/core asciidoctor-pdf@1.0.0-alpha.4
```

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "file-url": "3.0.0",
     "fs-extra": "7.0.1",
     "html-entities": "^1.2.1",
-    "mathjax": "3.0.0",
+    "mathjax": "3.0.1",
     "pagedjs": "0.1.35",
     "pdf-lib": "^1.2.1",
     "puppeteer": "2.0.0",

